src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetModuleHandle.cs

```diff
@@ -12,6 +12,6 @@ internal partial class Interop
     internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, EntryPoint = "GetModuleHandleW", CharSet = CharSet.Unicode, SetLastError = true)]
-        internal static extern IntPtr GetModuleHandle(string moduleName);
+        internal static extern IntPtr GetModuleHandle(string? moduleName);
```

Comment:
It seems like this file needs a #nullable enable entry since it is under the `Common` path.

Reply:
Yup. Thanks.